src/app/components/category-products/category-products.component.html

```diff
@@ -41,7 +41,7 @@ <h5 class="mb-3">Rating</h5>
 
                             <div class="form-check mb-2" *ngFor="let star_list of [].constructor(5); let i = index">
```

Comment:
czy jest może jakiś ładniejszy sposób na to żeby wygenerować te gwiazdki?
wiem, że moglabym wrzucic tą 5 do zmiennej i byloby troche ladniej, ale chodzi mi o cos jeszcze bardziej poprawnego